common/court_positions/types/00_court_positions.txt

```diff
@@ -744,7 +744,8 @@ garuda_court_position = {
 
 	# Is this an available position for this court? (root is court owner)
 	is_shown = {
-		culture = { has_cultural_parameter = unlock_garuda_warrior_title }
+		# Warcraft
+		# culture = { has_cultural_parameter = unlock_garuda_warrior_title }
```

Comment:
No requirements for it means that this title is available for everyone

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/173121255-b6ee69b7-6b80-4db7-9c6f-818f34769f0d.png)


</details>

Reply:
Addressed in 6a9914966